### PR TITLE
rss: Feature - Implement content-type exception handling/ignore.

### PR DESCRIPTION
**Pull request by valorcz**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by valorcz** [comment]
If I got it right, the failing check is actually complaining about the original structure of the RSS application, so it's above the scope of this feature pull request to fix it. But if needed, I could probably fix that too.

File: release_notes/unreleased.md [hunk 1]
```diff
@@ -1 +1,3 @@
 **Unreleased**
+
+* Added asset parameter 'ignore_cterrors'
```
Comment (dgopani-crest) [inline release_notes/unreleased.md:3]: Can we make it more user friendly

File: rss.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "product_version_regex": ".*",
     "publisher": "Splunk",
     "license": "Copyright (c) 2017-2022 Splunk Inc.",
-    "app_version": "2.0.4",
+    "app_version": "3.0.0",
```
Comment (dgopani-crest) [inline rss.json:13]: 2.1.0

**Review by ebhoir-crest — APPROVED** [review]

**Review by rgil-splunk — APPROVED** [review]

**Review by ricardo-phantom — APPROVED** [review]

File: rss_connector.py [hunk 3]
```diff
@@ -86,7 +90,9 @@ def is_positive_int(self, value):
             return False
 
     def _handle_test_connectivity(self, param):
-        return self.set_status_save_progress(phantom.APP_SUCCESS, rc.RSS_TEST_CONNECTIVITY_PASSED)
+        self.debug_print("In action handler for action: {}".format(self.get_action_identifier()))
```
[the remark this answers is not in the document]
Comment (pdesai-crest) [reply rss_connector.py:93]: Yes, as a workaround to the requirement of minimum 2 logging statements in every action

**Pull request merged**